keyboards/1upkeyboards/sweet16/keymaps/ridingintraffic/config.h

```diff
@@ -0,0 +1,19 @@
+#ifndef CONFIG_RIDINGINTRAFFIC
```

Comment:
If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?

Reply:
Basically, the file should look like this: 
```c
#pragma once

/* tap dance stuff*/
#undef TAPPING_TERM
#define TAPPING_TERM 500


#define EXAMPLESTRING1 "tapdance_1"
#define EXAMPLESTRING2 "tapdance_2"
#define EXAMPLESTRING3 "tapdance_3"
#define EXAMPLESTRING4 "tapdance_4"

#undef RGBLED_NUM
#define RGBLED_NUM 16
```